test/__snapshots__/matcher.test.ts.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I tried to use inline snapshots - but unfortunately, the whitespace is problematic, I think prettier may be reformatting the error messages to use spaces instead of tabs, which means the snapshots are broken on every test run.